### ISSUE TITLE
Remove "tabindex" attributes from cloned elements

### ISSUE THREAD

**Issue by ghost** [DataTables/FixedColumns#43]
When cloning elements, the "tabindex" attributes should be removed since it disrupts keyboard navigation.

**Comment by DataTables** [comment]
Good point - thanks! I'll look into this shortly.

**Comment by kim07** [comment]
:+1:

**Comment by DataTables** [comment]
Just being looking into this, and I'm not actually sure how to fix it. We want the tab index to remain on the floating elements (since they are the visible ones), but they are the clones. So I could set tabindex=-1 on the hidden elements, but then on the next clone it would copy the in correct tab index.

I could then have it set the found tab index to be 0, rather than -1, but that might be wrong... Storing the original tab index is really the only way to do that, but I'm concerned about performance impact.

This is going involve a bit more thought (a long term bug this one, sorry - it will be addressed when time is available!).

**Comment by thecristen** [comment]
Any update on this?

**Comment by DataTables** [comment]
No sorry. I've not had the change to look into the issue I raised above yet. Any input and PRs welcome :-)

**Comment by AllanJard** [comment]
Closing due to the v4 upgrade which significantly simplified FixedColumns through the use of `position sticky`. This removes the need for cloned elements and resolves a ton of issues, including this one.
